lib/matplotlib/mathtext.py

```diff
@@ -58,7 +59,7 @@ def __init__(self, output):
             {"path": "vector", "agg": "raster", "macosx": "raster"},
             output=output.lower())
 
-    def parse(self, s, dpi=72, prop=None):
+    def parse(self, s, dpi=72, prop=None, antialiased=None):
```

Comment:
As discussed on call:
```suggestion
    def parse(self, s, dpi=72, prop=None, *, antialiased=None):
```

Reply:
Thanks Elliott! It's done.